docs/starting-from-scratch.md

```diff
@@ -1,142 +1,449 @@
 ---
 id: starting-from-scratch
-title: Getting Started From Scratch
+title: Starting From Scratch
 sidebar_label: Starting from scratch
 ---
 
-A simple example of how to use webpack with [single-spa](https://github.com/joeldenning/simple-single-spa-webpack-example/blob/master/README.md). Check out the docs to learn more about single-spa itself.
+## Project Overview
+single-spa allows you to build micro frontends that coexist and can each be written with their own framework. This will allow you and your team to: 
 
-#### Note: You do not have to use webpack to use single-spa
-Since single-spa is capable of operating without using webpack, don't get too hung up on this. 
+1) Use multiple frameworks on the same page. [see the single-spa ecosystem for more info](https://single-spa.js.org/docs/ecosystem.html#docsNav)
+2) Write code using a new framework, without rewriting your existing application
+3) [Lazy load](https://en.wikipedia.org/wiki/Lazy_loading) code for improved initial load time
 
-Check out [this blog post]() to learn more.   
+For this tutorial, we will be focused on creating a one code repo, one build single-spa application from scratch using React and Angular1(AngularJS) with Webpack. 
```

Comment:
> creating a one code repo, one build single-spa application from scratch

That part ^ is a little confusing, since "repo" and "build" are nouns but are being used as adjectives. It wasn't clear to me how to interpret "one build single-spa application" when I first read it, but then I got it the second time.

> single-spa application

This ^ is a term that is defined and used very precisely in the documentation (see [here](https://github.com/CanopyTax/single-spa/blob/master/docs/applications.md#registered-applications)), and it is being used incorrectly in this sentence. A single-spa application refers to one of the many registered applications that compose the overall SPA. But here it is being used to refer to the overall SPA.

Here's a proposed alternative:
>Single-spa can be used with just about any build system or javascript framework, but this tutorial will focus on creating a web app with Webpack, React, and AngularJS. Our tutorial puts everything into a single code repository, but it is also possible to have separate code repositories for each of your applications.
>
> If you'd like to learn how to use single-spa with Angular 2+, Vue, or any other framework, try [these tutorials](https://sometutorial.com). And if you'd rather use a different build system instead of webpack, check out [this tutorial](https://sometutorial.com)